AK/Checked.h

```diff
@@ -129,37 +126,19 @@ class Checked {
         m_value = value;
     }
 
-    Checked(const Checked& other)
-        : m_value(other.m_value)
-        , m_overflow(other.m_overflow)
-    {
-    }
+    Checked(const Checked&) = default;
 
-    Checked(Checked&& other)
-        : m_value(exchange(other.m_value, 0))
-        , m_overflow(exchange(other.m_overflow, false))
-    {
-    }
+    Checked(Checked&& other) = default;
```

Comment:
This is a behavior change.

Reply:
Good catch.

Reply:
@awesomekling It appears that it only affects the "moved from" object's state. Since the "moved from" object should be treated as only eligible for assignment or destruction, why does this matter?